ConfigReader/ApplicationConfig.properties

```diff
@@ -0,0 +1,11 @@
+# This is file for containing different variables.
+
+URL=https://www.amazon.in/
+category=Amazon Fresh
+
+SenderAddress=ishita.nigam20052017@gmail.com
```

Comment:
Remove the sender address, password and recipient address from the properties file. These are considered to be private and sensitive information. They are not supposed to be exposed like this.

Reply:
Done.